llama_stack/apis/vector_dbs/vector_dbs.py

```diff
@@ -46,7 +46,7 @@ class VectorDBs(Protocol):
     @webmethod(route="/vector-dbs", method="GET")
     async def list_vector_dbs(self) -> ListVectorDBsResponse: ...
 
-    @webmethod(route="/vector-dbs/{vector_db_id}", method="GET")
```

Comment:
I'm having issues with `test_vector_io.py` and I think it may be related to this change.

Reply:
Confirmed it by reverting change.